### PR TITLE
Add a test and some refactoring

### PR DESCRIPTION
**Pull request by dsgoers**

### CODE REVIEW

File: spec/models/auth_token_spec.rb [hunk 1]
```diff
@@ -8,6 +8,11 @@
     assert(result == :created)
   end
 
+  it 'generates a value for the token' do
+    result = AuthToken.create(access_code: AccessCode.find(1))
+    assert(!result.nil?)
```
Comment (soberstadt) [inline spec/models/auth_token_spec.rb]: We prefer to use `expect` over `assert`, so this line should be `expect(result).to_not be nil`. I think that reads more naturally.

File: app/models/translation.rb [hunk 2]
```diff
@@ -24,7 +24,7 @@ def download_translated_page(page_filename)
                                     dev_hash: AuthHelper.dev_hash, locale: language.abbreviation,
                                     source_file_name: page_filename, export_file_name: page_filename }
   rescue RestClient::ExceptionWithResponse => e
-    return e.response
+    raise e
```
Comment (soberstadt) [inline app/models/translation.rb]: Now this becomes an unnecessary rescue too.

File: spec/s3_helper_spec.rb [hunk 3]
```diff
@@ -8,12 +8,12 @@
     push
 
     pages_dir = Dir.glob('pages/*')
-    assert(pages_dir.empty?)
+    expect(pages_dir.empty?).to be(true)
```
Comment (soberstadt) [inline spec/s3_helper_spec.rb]: When learning rspec, one of the lessons is that you (usually) don't have to do `.to be(true)` or `.to be(false)`.
Instead `expect(pages_dir).to be_empty` (ruby will read `be_xxx` and try `xxx?`)

Read more here: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers

File: spec/s3_helper_spec.rb [hunk 4]
```diff
@@ -8,12 +8,12 @@
...
   end
 
   it 'deletes zip file' do
     push
-    assert(!File.exist?('version_1.zip'))
+    expect(File.exist?('version_1.zip')).to be(false)
```
Comment (soberstadt) [inline spec/s3_helper_spec.rb]: `expect(File).to_not exist('version_1.zip')`

**Comment by dsgoers** [comment]
tanks

**Pull request merged**